Makefile

```diff
@@ -0,0 +1,138 @@
+all:				hello
+
+.SECONDARY:
+
+version	=			'1.53'
+date =				'2016-11-29'
+
+hello:
+	@echo -e '\e[32mThis is DeGeŠ Makefile, version \e[95m$(version)\e[32m [\e[95m$(date)\e[32m]\e[0m'
```

Comment:
Napíšem to iba tu, ale platí to pre celý projekt: VT-100 kódy takýmto spôsobom priamo do stringov nepatria. Priznám sa že neviem aký je vzorák, ale existuje binárka `tput` ktorá si zistí typ terminálu a mala by urobiť správnu vec (napríklad `tput setaf 2` vypľuje znak ktorý nastaví terminál do zelenej farby).
Potom si vieš nastaviť napr premennú `green := $(shell tput setaf 2)`, `norm := $(shell tput sgr0)` a napísať
`@echo -e '$(green)Toto je zelené$(norm), toto už nie`. A ideálne tie premenné nastaviť kondicionálne aby sa farebný output dal vypnúť úplne nejakou env var.